common/app/common/errors/ErrorsBean.java

```diff
@@ -7,21 +7,28 @@
 import java.util.Collections;
 import java.util.List;
 
+import static java.util.Collections.singletonList;
+
 public class ErrorsBean extends Base {
     private List<ErrorBean> globalErrors;
 
     public ErrorsBean() {
     }
 
     public ErrorsBean(final String errorMessage) {
-        this.globalErrors = Collections.singletonList(new ErrorBean(errorMessage));
+        this.globalErrors = singletonList(new ErrorBean(errorMessage));
     }
 
-    public ErrorsBean(final Form<?> filledForm) {
+    public ErrorsBean(final List<Form<?>> filledForms) {
```

Comment:
Because initially I was sending the billingAddress and shippingAddress together and showing all the errors. But there could be too many and I thought better to send them separately, so I don't use the list of forms anymore. I thought about leaving it for the future, as it doesn't have the easiest code, to avoid losing it for the future.